src/single-spa-react.test.js

```diff
@@ -372,6 +373,63 @@ describe("single-spa-react", () => {
         });
     });
 
+    it(`should not log a warning when errorBoundary opts is passed in`, () => {
+      const props = { why: "hello", name: "hi" };
+      const lifecycles = singleSpaReact({
+        React,
+        ReactDOM,
+        rootComponent: function foo() {},
+        errorBoundary() {
+          return null;
+        },
+      });
+
+      return lifecycles
+        .bootstrap()
+        .then(() => lifecycles.mount(props))
+        .then(() => {
+          return expect(console.warn).not.toHaveBeenCalled();
+        });
+    });
+
+    it(`should call opts.errorBoundary during an error boundary handler`, () => {
```

Comment:
This test has me feeling queasy - we should consider switching to `@testing-library/react`